lib/bundler/installer/gem_installer.rb

```diff
@@ -16,7 +16,7 @@ def install_from_spec
       Bundler.ui.debug "#{worker}:  #{spec.name} (#{spec.version}) from #{spec.loaded_from}"
       generate_executable_stubs
       return true, post_install_message
-    rescue Bundler::InstallHookError, Bundler::SecurityError
+    rescue Bundler::InstallHookError, Bundler::SecurityError, APIResponseMismatchError
```

Comment:
they could, just didn't feel like changing the other ones since they were already there